22:46:10,515 WARN  eu.stratosphere.nephele.configuration.GlobalConfiguration     - Cannot load configuration: unknown element plugins
22:46:10,552 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Reading location of job manager from configuration
22:46:10,561 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Determined address of job manager to be cloud-11.dima.tu-berlin.de/130.149.21.15:6002
22:46:10,569 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Announcing connection information cloud-23 to job manager
22:46:10,634 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server Responder: starting
22:46:10,634 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server listener on 6122: starting
22:46:10,635 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server handler 0 on 6122: starting
22:46:11,750 INFO  eu.stratosphere.nephele.taskmanager.bufferprovider.GlobalBufferPool  - Initialized global buffer pool with 32768 buffers with a size 32768 bytes each
22:46:11,769 INFO  eu.stratosphere.nephele.taskmanager.bytebuffered.ByteBufferedChannelManager  - Initialized byte buffered channel manager with sender-side spilling disabled
22:46:11,779 INFO  eu.stratosphere.nephele.instance.HardwareDescriptionFactory   - Found Tenured Gen pool (max: 24051843072, used: 0)
22:46:11,779 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Initializing memory manager with 22528 megabytes of memory
22:46:46,460 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Build Triads (16/25)
22:46:46,463 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Build Triads (16/25)
22:46:48,537 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (122/200)
22:46:48,538 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (122/200)
22:46:48,540 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (122/200)
22:46:48,541 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (122/200)
22:46:49,885 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (126/200)
22:46:49,886 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (126/200)
22:46:49,888 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (126/200)
22:46:49,889 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (126/200)
22:46:50,355 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (125/200)
22:46:50,356 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (125/200)
22:46:50,357 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (125/200)
22:46:50,358 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (125/200)
22:46:50,660 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (123/200)
22:46:50,661 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (123/200)
22:46:50,662 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (123/200)
22:46:50,663 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (123/200)
22:46:51,132 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (121/200)
22:46:51,132 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (121/200)
22:46:51,134 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (121/200)
22:46:51,135 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (121/200)
22:46:51,800 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (128/200)
22:46:51,800 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (128/200)
22:46:51,802 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (128/200)
22:46:51,804 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (128/200)
22:46:52,641 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (124/200)
22:46:52,641 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (124/200)
22:46:52,643 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (124/200)
22:46:52,644 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (124/200)
22:46:52,914 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (127/200)
22:46:52,915 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (127/200)
22:46:52,916 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (127/200)
22:46:52,917 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (127/200)
22:47:19,731 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Finished PACT code.: Build Triads (16/25)
22:47:19,731 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from RUNNING to FINISHING for task Build Triads (16/25)
22:49:25,247 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-1938759838222704517_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.27:45961 remote=/130.149.21.27:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:25,248 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1938759838222704517_1722 bad datanode[0] 130.149.21.27:45020
22:49:25,249 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1938759838222704517_1722 in pipeline 130.149.21.27:45020, 130.149.21.22:45020, 130.149.21.17:45020: bad datanode 130.149.21.27:45020
22:49:26,605 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-6704960506959575861_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.27:45948 remote=/130.149.21.27:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:26,606 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-6704960506959575861_1722 bad datanode[0] 130.149.21.27:45020
22:49:26,607 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-6704960506959575861_1722 in pipeline 130.149.21.27:45020, 130.149.21.28:45020, 130.149.21.65:45020: bad datanode 130.149.21.27:45020
22:49:27,546 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_697379904306874172_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.27:45959 remote=/130.149.21.27:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:27,547 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_697379904306874172_1722 bad datanode[0] 130.149.21.27:45020
22:49:27,548 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_697379904306874172_1722 in pipeline 130.149.21.27:45020, 130.149.21.30:45020, 130.149.21.22:45020: bad datanode 130.149.21.27:45020
22:49:29,561 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_9139684618760994224_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.27:45963 remote=/130.149.21.27:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:29,562 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_9139684618760994224_1722 bad datanode[0] 130.149.21.27:45020
22:49:29,563 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_9139684618760994224_1722 in pipeline 130.149.21.27:45020, 130.149.21.66:45020, 130.149.21.34:45020: bad datanode 130.149.21.27:45020
22:49:41,202 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_2859398990846706985_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.27:45968 remote=/130.149.21.27:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:41,204 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_2859398990846706985_1722 bad datanode[0] 130.149.21.27:45020
22:49:41,204 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_2859398990846706985_1722 in pipeline 130.149.21.27:45020, 130.149.21.33:45020, 130.149.21.17:45020: bad datanode 130.149.21.27:45020
22:49:49,766 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_5374982765539209272_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.27:45949 remote=/130.149.21.27:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:49,767 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_5374982765539209272_1722 bad datanode[0] 130.149.21.27:45020
22:49:49,768 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_5374982765539209272_1722 in pipeline 130.149.21.27:45020, 130.149.21.64:45020, 130.149.21.25:45020: bad datanode 130.149.21.27:45020
22:50:09,295 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_1591257252091676750_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.27:45970 remote=/130.149.21.27:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:09,296 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_1591257252091676750_1722 bad datanode[0] 130.149.21.27:45020
22:50:09,296 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_1591257252091676750_1722 in pipeline 130.149.21.27:45020, 130.149.21.18:45020, 130.149.21.2:45020: bad datanode 130.149.21.27:45020
22:50:16,535 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_2116783748551405129_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.27:45952 remote=/130.149.21.27:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:16,536 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_2116783748551405129_1722 bad datanode[0] 130.149.21.27:45020
22:50:16,537 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_2116783748551405129_1722 in pipeline 130.149.21.27:45020, 130.149.21.60:45020, 130.149.21.63:45020: bad datanode 130.149.21.27:45020
22:50:31,346 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-1938759838222704517_1740java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.27:43489 remote=/130.149.21.22:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:31,347 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1938759838222704517_1740 bad datanode[0] 130.149.21.22:45020
22:50:31,347 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1938759838222704517_1740 in pipeline 130.149.21.22:45020, 130.149.21.17:45020: bad datanode 130.149.21.22:45020
22:50:32,708 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-6704960506959575861_1747java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.27:36539 remote=/130.149.21.28:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:32,709 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-6704960506959575861_1747 bad datanode[0] 130.149.21.28:45020
22:50:32,709 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-6704960506959575861_1747 in pipeline 130.149.21.28:45020, 130.149.21.65:45020: bad datanode 130.149.21.28:45020
22:50:33,688 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_697379904306874172_1753java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.27:55702 remote=/130.149.21.30:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:33,689 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_697379904306874172_1753 bad datanode[0] 130.149.21.30:45020
22:50:33,690 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_697379904306874172_1753 in pipeline 130.149.21.30:45020, 130.149.21.22:45020: bad datanode 130.149.21.30:45020
22:50:35,660 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_9139684618760994224_1764java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.27:45122 remote=/130.149.21.66:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:35,661 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_9139684618760994224_1764 bad datanode[0] 130.149.21.66:45020
22:50:35,662 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_9139684618760994224_1764 in pipeline 130.149.21.66:45020, 130.149.21.34:45020: bad datanode 130.149.21.66:45020
22:50:47,301 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_2859398990846706985_1790java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.27:38913 remote=/130.149.21.33:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:47,302 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_2859398990846706985_1790 bad datanode[0] 130.149.21.33:45020
22:50:47,303 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_2859398990846706985_1790 in pipeline 130.149.21.33:45020, 130.149.21.17:45020: bad datanode 130.149.21.33:45020
22:50:55,863 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_5374982765539209272_1800java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.27:54425 remote=/130.149.21.64:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:55,865 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_5374982765539209272_1800 bad datanode[0] 130.149.21.64:45020
22:50:55,865 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_5374982765539209272_1800 in pipeline 130.149.21.64:45020, 130.149.21.25:45020: bad datanode 130.149.21.64:45020
22:51:15,393 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_1591257252091676750_1847java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.27:52252 remote=/130.149.21.18:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:51:15,394 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_1591257252091676750_1847 bad datanode[0] 130.149.21.18:45020
22:51:15,395 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_1591257252091676750_1847 in pipeline 130.149.21.18:45020, 130.149.21.2:45020: bad datanode 130.149.21.18:45020
22:51:22,634 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_2116783748551405129_1874java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.27:56307 remote=/130.149.21.60:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:51:22,635 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_2116783748551405129_1874 bad datanode[0] 130.149.21.60:45020
22:51:22,636 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_2116783748551405129_1874 in pipeline 130.149.21.60:45020, 130.149.21.63:45020: bad datanode 130.149.21.60:45020
